proto/internal/temporal/server/api/schedule/v1/message.proto

```diff
@@ -155,3 +159,55 @@ message HsmSchedulerState {
     google.protobuf.Timestamp next_invocation_time = 3;
 
 }
+
+// State machine scheduler internal state.
+message SchedulerInternal {
+    temporal.server.api.enums.v1.Scheduler2State state = 1;
+
+    // Scheduler request parameters and metadata. 
+    temporal.api.schedule.v1.Schedule schedule = 2;
+    temporal.api.schedule.v1.ScheduleInfo info = 3;
+    temporal.api.schedule.v1.SchedulePatch initial_patch = 4;
```

Comment:
Hmm... why is this the initial patch? Where do we store the "latest" patch? Is that required? I'm not super familiar with the business logic here.